questionary/prompts/common.py

```diff
@@ -151,7 +152,17 @@ def __init__(
         self.use_pointer = use_pointer
         self.default = default
 
-        self.pointed_at = None
+        if initial_choice is not None:
+            if initial_choice in choices:
+                self.pointed_at = choices.index(initial_choice)
```

Comment:
do we need to check if this choice can actually be selected (as in: making sure it is not disabled)?

Reply:
@tmbo This is done here:

https://github.com/tmbo/questionary/pull/56/files#diff-0eccf09aa36dc0dd59d841a342a6b76bR175-R179

Reply:
:+1: